packages/base/src/widget.ts

```diff
@@ -114,9 +114,11 @@ export class JupyterCadPanel extends SplitPanel {
   }) {
     this._view = new ObservableMap<JSONValue>({});
     const cameraSettings: CameraSettings = { type: 'Perspective' };
+    const axes: AxeHelper = { visible: false, size: 20 };
```

Comment:
Let's remove the `AxeHelper.size` property from the type since we won't be using it anymore

Reply:
@martinRenou Thanks for the comment. `size` has been removed from `AxeHelper` type.